wire-desktop-beta/wire-desktop-beta.desktop

```diff
@@ -1,11 +1,11 @@
 [Desktop Entry]
 Type=Application
 Version=1.1
-Name=Wire
+Name=Wire (beta)
 GenericName=Secure messenger
 Comment=Secure messenger for everyone
-Exec=wire-desktop
-Icon=wire-desktop
+Exec=wire-desktop-beta
```

Comment:
I acknowledge that executable file name changed, but I feel it's reasonable. I believe other packages like Firefox Beta also provide a different executable name and don't conflict with "base" package.

I assume most people start Wire via some kind of launcher anyway, where now the distinction is quite clear:

![image](https://user-images.githubusercontent.com/1177900/50539515-d4c6dd00-0b81-11e9-8971-30db6f7e7283.png)